homeassistant/components/tractive/__init__.py

```diff
@@ -124,16 +128,24 @@ async def _listen(self):
                     if event["message"] == "activity_update":
                         self._send_activity_update(event)
```

Comment:
I think we can use `continue` instead of `else` and reduce the indentation of the following lines.
```suggestion
                        self._send_activity_update(event)
                        continue
```